internal/devpkg/package.go

```diff
@@ -830,3 +830,15 @@ func packageInstallErrorHandler(err error, pkg *Package, installableOrEmpty stri
 
 	return usererr.WithUserMessage(err, "error installing package %s", pkg.Raw)
 }
+
+func (p *Package) ResolvedVersion() (string, error) {
+	if err := p.resolve(); err != nil {
+		return "", err
+	}
```

Comment:
If package is not in lockfile this is just current resolved version, but that's probably ok.